src/index.ts

```diff
@@ -7,4 +7,5 @@ interface TimeZone {
   utc: string;
 }
 
+export { TimeZone };
```

Comment:
@kataik It should be better if you put `export` directly to the interface: `export interface TimeZone { ...`

Reply:
@duxor I don't have any hard feelings either way, just curious what difference does it make? I personally find having all imports at the top and all exports at the bottom easier to  read, but I understand that's preference. Based on your comment, I assume there's a reason why exports directly on the declarations are better?

Reply:
@kataik yep, I agree that we need to keep imports at the top, and exports at the bottom, but there is no reason to have separated export if we can use it in the declaration - one code line less will be readable, in my opinion.  

Of course, that is just a convention, and there are a lot of discussions on the topic :D 

Some resources:
https://github.com/airbnb/javascript/issues/1275
https://github.com/airbnb/javascript#modules--use-them
https://stackoverflow.com/questions/60106063/export-an-es6-default-class-inline-with-definition-or-at-end-of-file
https://github.com/import-js/eslint-plugin-import/blob/main/docs/rules/exports-last.md

Reply:
@duxor sorry for disagreeing, but having inline exports is not a convention, more like a preference, because readability is subjective. I personally never like to assume that there will be no more exports later down the road, in which case I find what is actually exported harder to find in the source if they are defined inline (basically I find it less readable). None of the links you provided above are declaring or even suggesting that inline exports (in any case) would be a convention. In my teams we use airbnb's linter configuration as-is and it allows non-inline exports. So it's definitely not going against any style guides. My preference is to keep it as-is, in my opinion sparing one line of code in a less than a page long source file is meaningless from a readability PoV whereas always keeping the exports as a single block helps to immediately understand what's exported by the module when you look at it - which is a way more common action for developers than to actually read the implementation itself. It's a kind of separation of concerns in my book, the act of exporting is more cohesive with other acts of exporting than they are with what they are exporting exactly.

Reply:
having said that, happy to disagree and commit, as I said, I don't have strong feelings about it

Reply:
@kataik Yep, but what is the convention? In my point of view, those are preferences that we want to implement in the project. 😄 

So, thank you for a great discussion, we have different opinions and that is ok. The repository owner should decide which approach should be a convention in this case.

Reply:
I agree with @kataik . Thank you @duxor for the review.